### DISCUSSION TITLE
Deband/Dither/Noise filter

### DISCUSSION THREAD

**Discussion by thehhmdb** [alvr-org/ALVR#1559] [General]
Hi, 

Would it be possible to add a Deband/Dither/Noise filter to help with the banding especially on h264?

Example of banding (image 1) and debanding (image 2):
![image](https://user-images.githubusercontent.com/68663972/230629572-9a689a44-dabe-4052-804b-cb7d5a5cd498.png)
![image](https://user-images.githubusercontent.com/68663972/230629601-484c7159-f97c-43e4-a796-29ad05265a38.png)
https://rigaya34589.blog.fc2.com/blog-entry-937.html

It could help improve some of the compression artifacts people associate with encoded VR video streams

Thanks

**Comment by zmerp** [comment]
That's possible, yeah. Maybe I could work on it in the future but I'm not sure I'll have the chance.
